### DISCUSSION TITLE
Create Table issue

### DISCUSSION THREAD

**Discussion by iv-max** [dbeaver/dbeaver#22966] [Q&A] [answered]
### Description

Hi!
I'm student in a college and one of our assignments was to create a simple table, but something went wrong.
I wrote an query bellow, I've got different errors trying to execute it.
But, I've easily executed it on MS SQL Server Management Studio.
Actually, I've got 3 different error messages from DBeaver trying to execute this query, those are: 
incorrect syntax near CONSTRAINT, incorrect syntax near 2, incorrect syntax near )  

So, if I'm doing something wrong, please let me know, I've tryed to troubleshoot myself, but I did't found the mistake.

Here's the query:

CREATE Table CoinCollection
(
coinID int identity (1000,1) Primary Key,

coinType char(2) not null default 'P',
productionDate datetime not null,
coinMint char(2) not null,
coinCondition char(2) not null,
coinValue money,
appraisingDate datetime,
comments nvarchar(255),

CONSTRAINT ChkTypeMintCond 
CHECK (
coinType in('P', 'N', 'D', 'Q', 'H1', '1')
AND
coinMint in ('D', 'P', 'S', 'O', 'CC')
AND
coinCondition in ('PR', 'BU', 'UC', 'VF', 'F', 'G', 'P') 
)
)

### DBeaver Version

Enterprise 23.3

### Operating System

mac sonoma 14.3

### Database and driver

1. MS SQL server
2. jdbc

### Steps to reproduce

Just try to execute query from description

### Additional context

VPN is used to connec to to the college server.

**Comment by E1izabeth** [comment]
I successfully created table by executing the provided DDL.
Could you provide some footage of your issue?

**Comment by iv-max** [comment]
Thanks for quick response! 
I've checked again, and I've dicovered that error depends on cursor position and selection. So if I rundomly set cursor on any line, or last line, or extra empty line. The execute. Execution works like one line was selected (the one where cursor was just 'blinking'.
And, when I selected the whole query it was executed as expected.
Is it normal behavior?

**Comment by E1izabeth** [comment, marked as answer]
You have empty lines in your query. In DBeaver empty line is treated as a statement end by default.
Turning off this setting or removing empty lines will help you
![image](https://github.com/dbeaver/dbeaver/assets/28875055/92a29d4a-16df-4128-bdcf-8a36e7d71ca4)

**Comment by iv-max** [comment]
Thank you!!!
